sample01.go

```diff
@@ -1,7 +1,19 @@
 package sample01
 
+import "fmt"
+
+type Connpass struct {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
exported type Connpass should have comment or be unexported